test/ffi/mk_event_create.cpp

```diff
@@ -42,3 +42,32 @@ TEST_CASE("mk_event_create_() deals with invalid JSON event key") {
   REQUIRE(ev != nullptr);
   REQUIRE(mk_event_serialization(ev.get()) != nullptr);
 }
+
```

Comment:
These changes are to make sure that we correctly deal with U+0000 and UTF-8 characters.